### PR TITLE
[Chartboost] Release SDK 9.0.0

### PR DESCRIPTION
**Pull request by stosiarz**
The Chartboost Android SDK 9.0.0 has been updated.

Release Notes
  Improvements:
  - Implemented new public API
  - Separated ad types from main Chartboost API
  - Improved stability of the SDK

  Fixes:
  - Small quality of life improvements and fixes for pre-cache video ads
 
  Notes:
  - Sdk is compatible with Android API level 32

### CODE REVIEW

**Comment by thomasmso** [comment]
@stosiarz - We'll check this out and prioritize for the next couple weeks

File: Chartboost/src/main/java/com/applovin/mediation/adapters/ChartboostMediationAdapter.java [hunk 1]
```diff
@@ -509,100 +516,179 @@ public void didDismissRewardedVideo(String location)
...
+                    {
+                        ChartboostMediationAdapterRouter.this.onAdLoaded( location );
+                    }
+
+                    showAdViewDelayed();
+                }
+            }
+
+            @Override
+            public void onAdClicked(@NonNull ClickEvent clickEvent, @Nullable ClickError clickError) {
```
Comment (stosiarz) [inline Chartboost/src/main/java/com/applovin/mediation/adapters/ChartboostMediationAdapter.java:687]: To the reviewer: During internal tests we found odd behaviour. In the adapter we reach banner onAdClick callback and we call ChartboostMediationAdapterRouter.this.onAdClicked( location ); but then this callback is not forwarded back to the caller which is applovin MaxAdView and listener implemented is: override fun onAdClicked(ad: MaxAd?)

Is this a bug or something wrong with our implementation? Interstitial and Rewarded forwards the callbacks as expected upon click.

Comment (thomasmso) [reply Chartboost/src/main/java/com/applovin/mediation/adapters/ChartboostMediationAdapter.java:687]: It's your implementation - we have a router architecture specifically for your SDK due to the singleton callback but now you've moved over to a new set of more deterministic APIs. We're working/testing this on our end accordingly.

**Comment by Billy-Hu** [comment]
Changes merged, adapter 9.0.0.0 is not live, commit:
https://github.com/AppLovin/AppLovin-MAX-SDK-Android/commit/a73ddb2df4d63f73c78216e2bab59493e685d915

**Pull request closed without merging**